src/main/markdown/eco/application.md

```diff
@@ -0,0 +1,34 @@
+# Application Frameworks 
+
+One of the main tasks, when writing a GWT application is separating the code and managing the state. GWT offers with
+[MVP Activities and Places](/doc/latest/DevGuideMvpActivitiesAndPlaces.html) modules to do that. Over the
+time other implementations evolved. 
+
+## domino-mvp and domino-historie<a id="domino-mvp-historyt"></a>
+
+Domino-mvp is a small, simple and a very lightweight framework for building both simple and complex applications using
+GWT, it introduce the concept of routing using both tokens and events, also allows the developers to write a modular 
+application and shared components with any other domino application.
+
+For more information about domino-mvp, see: [domino-mvp at GitHub](https://github.com/DominoKit/domino-mvp)
+
+Domino-historie is a wrapper for the browser history state API.
+
+For more information about domino-history, see: [domino-history at GitHub](https://github.com/DominoKit/domino-history)
+
+## Nalu<a id="nalu"></a>
+
+Nalu is a tiny framework that helps you to create GWT based applications quite easily. Using the HTML 5 history for 
+routing and navigation, Nalu supports the browser's back-, forward-, and reload-button by default and without any
+need to implement anything.
+
+For more information about domino-history, see: [Nalu at GitHub](https://github.com/NaluKit/nalu)
+
+## Support
+
+The GWT project team does not support this projects. In case you have questions or would like to open an issue, please
+contact the contributor of the project!
+
+## Missing Something?
+
+If you are missing something, following the instructions located [here](add-lib.html).
```

Comment:
Maybe we phrase this as *If you are missing a library or a framework from the list, please follow the instructions here to add it.